stack/base/Dockerfile

```diff
@@ -42,8 +42,19 @@ RUN cat /opt/requirements.txt | xargs -I{} conda config --system --add pinned_pa
 
 # Configure pip to use the same requirements file as constraints file.
 ENV PIP_CONSTRAINT /opt/requirements.txt
-# Ensure that pip installs packages to ~/.local by default
-COPY pip.conf /etc/pip.conf
+# Ensure that pip installs packages to '~/.local/lib/python3.X/site-packages/' by default
```

Comment:
Is this means when python version changed in the image, dependencies of apps need to be reinstalled?

Reply:
Indeed, see #317